Dockerfile.build

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13
+FROM golang:1.17.13
```

Comment:
Any reason not to use golang 1.19?

Reply:
No I just picked that one to be on par with PX...